easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = "IOR"
+version = "3.0.1" 
+
+homepage = 'https://github.com/chaos/ior' 
+description = """ The IOR software is used for benchmarking parallel file systems 
+using POSIX, MPIIO, or HDF5 interfaces. """
+
+toolchain = {'name': 'foss', 'version': '2016a' }
+
+source_urls = [('https://github.com/chaos/ior/archive/')]
+sources = ['%(version)s.tar.gz']
+
+# gmake posix      -- IOR with only POSIX interface
+# gmake mpiio      -- IOR with only POSIX and MPIIO interfaces
+# gmake hdf5       -- IOR with POSIX, MPIIO, and HDF5 interfaces
+# gmake ncmpi      -- IOR with POSIX, MPIIO, and NCMPI interfaces
+# gmake all        -- IOR with POSIX, MPIIO, HDF5, and NCMPI interfaces
+preconfigopts = "./bootstrap &&" 
+#buildopts = ' mpiio'
```

Comment:
@RvDijk did the build procedure change?

If so, please just drop this line, and the comments outlining the options above?